README.md

```diff
@@ -23,10 +71,59 @@ Additionally, Kuadrant provides the following CRDs
 | [Limitador CRD](doc/ratelimitpolicy-reference.md)                                                            | [Limitador Operator](https://github.com/Kuadrant/limitador-operator) | Represents an instance of Limitador | [Limitador CR](https://github.com/Kuadrant/limitador-operator/blob/main/config/samples/limitador_v1alpha1_limitador.yaml)         |
 | [Authorino CRD](https://github.com/Kuadrant/authorino-operator#the-authorino-custom-resource-definition-crd) | [Authorino Operator](https://github.com/Kuadrant/authorino-operator) | Represents an instance of Authorino | [Authorino CR](https://github.com/Kuadrant/authorino-operator/blob/main/config/samples/authorino-operator_v1beta1_authorino.yaml) |
 
+<img alt="Kuadrant Architecture" src="doc/images/kuadrant-architecture.svg" />
+
+## Getting started
+
+### Pre-requisites
+
+* Istio is installed in the cluster. Otherwise, refer to the
+  [Istio getting started guide](https://istio.io/latest/docs/setup/getting-started/).
+* Kubernetes Gateway API is installed in the cluster. Otherwise,
+  [configure Istio to expose a service using the Kubernetes Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/).
+* Kuadrant is installed in the cluster.
```

Comment:
This is "wrong". Kuadrant needs to be installed in the cluster, true, but this guide should show how.

We need a guide to deploy kuadrant. Do we have one or needs to be done? currently it is only creating the kuadrant CR in a given namespace.

I would say the getting started section is:
* Pre-requisites
* Installing kuadrant
* Protect your service
  * If you are an *API Provider*
  * If you are a *Cluster Operator*

Reply:
You're right, I'll write the missing parts to make more sense